core/auth/webidentity.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"context"
+
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type (
+	// RequestIdentifier resolves an identity from a web request
+	RequestIdentifier interface {
+		Identify(ctx context.Context, request *web.Request) Identity
```

Comment:
unauthenticated request would return nil?

That would then in future be the way how to check for logged in or not?

Reply:
Right now: yes, though we could return an error or bool here too?
Also for guests: we can return a GuestIdentity or nil.

Reply:
As I remember - idea was to have role provider in auth module, which would add PermissionAuthorised (or similar), and base on that we should now if user is logged in or not -> so, from that perspective, checking if Identity is nil or by additional error/bool value, would be same :)

Reply:
Yes, security has the same, i might remove some here